src/components/TertiaryButton/TertiaryButton.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen, axe } from 'testUtils';
+
+import TertiaryButton from './TertiaryButton';
+
+test('renders correctly', () => {
+  render(<TertiaryButton>Hello world</TertiaryButton>);
+
+  expect(
+    screen.getByRole('button', { name: 'Hello world' }),
+  ).toBeInTheDocument();
+});
+
+test('has no accessibility errors', async () => {
+  const { container } = render(<TertiaryButton>Hello world</TertiaryButton>);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+test('renders with left alignment', () => {
+  render(<TertiaryButton align="left">Hello world</TertiaryButton>);
+  expect(screen.getByRole('button', { name: 'Hello world' })).toHaveStyle(
+    'text-align: left',
+  );
+});
+
+test('renders with center text alignment', () => {
+  render(<TertiaryButton align="center">Hello world</TertiaryButton>);
+  expect(screen.getByRole('button', { name: 'Hello world' })).toHaveStyle(
+    'text-align: center',
+  );
+});
+
+test('disabled renders correctly', () => {
+  render(<TertiaryButton disabled>Hello world</TertiaryButton>);
+  expect(
+    screen.getByRole('button', { name: 'Hello world' }).closest('button'),
```

Comment:
```suggestion
    screen.getByRole('button', { name: 'Hello world' }),
```
don't need `closest` anymore?